managed/src/main/java/com/yugabyte/yw/controllers/CertificateController.java

```diff
@@ -145,9 +146,32 @@ public Result get(UUID customerUUID, String label) {
     }
   }
 
+  public Result delete(UUID customerUUID, UUID reqCertUUID) {
+    CertificateInfo certificate = CertificateInfo.get(reqCertUUID);
+    if (certificate == null) {
+      return ApiResponse.error(BAD_REQUEST, "Invalid certificate.");
+    }
+    if (!certificate.customerUUID.equals(customerUUID)) {
+      return ApiResponse.error(BAD_REQUEST, "Certificate doesn't belong to customer");
+    }
+    if (!certificate.getInUse()) {
+      if (certificate.delete()) {
+        ObjectNode responseJson = Json.newObject();
+        responseJson.put("Successfully deleted the certificate", true);
```

Comment:
Any reason we need the message here? Can't we just return the success as empty? Doesn't it implicitly mean the task was successful?

Reply:
Nope , we are not showing it in FE . I can remove this.

Reply:
Done